funnel/templates/event.html.jinja2

```diff
@@ -1,7 +1,7 @@
 {% extends "layout.html.jinja2" %}
 {% from "macros.html.jinja2" import rsvpform, rsvpscript %}
 {% from "baseframe/forms.html.jinja2" import renderfield %}
-{% block title %}{{ project.title }}{% endblock %}
+{% block title %}Setup events{% endblock %}{% block subtitle %} — {{ project.title }}{% endblock %}
```

Comment:
Found it. In the beginning of each of these templates, add after the extends tag:

```jinja2
{% set subtitle = "whatever" %}
```

And in `layout.html.jinja2`'s `titletags` block, simply use `{% if subtitle %} – {{ subtitle }}{% endif %}`

I think we should call this something other than subtitle, because it's likely to clash with the app's use of subtitles for blurbs (later if not now). This could be `title_suffix` or `title_ext`.